### PR TITLE
Remove FrontPageMain and move logic to /pages

### LINKED ISSUE #5169: Remove FrontPageMain, and move logic to /pages
**Is your feature request related to a problem? Please describe.**
We are removing the 'Main' containers across the board, and splitting up our page level containers into one 'page' per page type. You can see an example of the work that was done for CPS Asset pages here: https://github.com/bbc/simorgh/pull/5166. This is the first step in a broader set of changes to avoid branching based on page type within containers, make it clearer what the concept of a 'container' is in Simorgh, and to move processing out of our components.

We have moved all of the Page containers into a new directory, /pages. As well as the Page containers, we also currently have a ‘Main’ container for each page type, which are still in the containers folder. For example, Article and ArticleMain, FrontPage and FrontPageMain, RadioPage and RadioPageMain. We also had one container for a CPS asset, as well as its corresponding ‘Main’ container. When using this approach, the Page container just wrapped the ‘Main’ container with the higher order components that were being used to enhance each page.

We have now decided that we should have a Page container for each CPS asset type, such as CpsPgl, CpsSty, and CpsMap. This is because the pages are different enough that we would have needed lots of branching logic within a generic CPS container, and it would have got messy and difficult to reason about. So, in the /pages directory, we have created these.

We also decided that it wasn’t really useful to have the container in the /pages directory that didn’t do anything other than wrap a 'Main' container with the higher order components - each of these containers was doing exactly the same thing. So we decided to move all of the logic from the 'Main' container into its corresponding top level container in the /pages directory, and delete the 'Main' containers. Now, for each page type, you can find the container for that page type in the /pages directory, and see all of the logic for that page type in one place. If we introduce a new page type, we add a container for it in the /pages directory.

We have already used this approach for the CpsPgl, CpsSty, and CpsMap pages, and we have issues raised to do this for all of the other page types. This is step one of a few tasks to clean up our containers, make the distinction between a ‘component’ and a ‘container’ clearer, and rethink how we process data in Simorgh to move it away from various levels of the component tree (more info is here: https://github.com/bbc/simorgh/issues/3962)

**Describe the solution you'd like**
Move the logic and tests from the FrontPageMain container to the FrontPage page, following the approach of the pull request referenced above.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #5169

**Overall change:** 
Move the logic and tests from the `FrontPageMain` container to the `FrontPage` page.

**Code changes:**
- Update `/pages/FrontPage` 
- Update `FrontPage` stories
- Update `FrontPage` tests
- Remove `FrontPageMain` container

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added labels to this PR for the relevant pod(s) affected by these changes
- [X] I have assigned this PR to the Simorgh project

**Testing:**
- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by sareh** [comment]
Blocking this on the Desktop layout work being merged, since this will cause conflicts and we want to minimise issues since Desktop should be merged tomorrow, pending going live on Wednesday.

File: src/app/pages/FrontPage/index.test.jsx [hunk 1]
```diff
@@ -31,7 +64,7 @@ jest.mock('../../containers/PageHandlers/withVariant', () => Component => {
 jest.mock('../../containers/PageHandlers/withContexts', () => Component => {
   const DataContainer = props => (
     <div id="ContextsContainer">
-      <Component {...props} />
+      <Component {...props} />{' '}
```
Comment (thekp) [inline src/app/pages/FrontPage/index.test.jsx]:
```suggestion
      <Component {...props} />
```

**Review by thekp — APPROVED** [review]
Looks good, the forecast says conflicts are coming. ⛈

**Review by EinsteinNjoroge — APPROVED** [review]
Looks good

**Review by thekp — APPROVED** [review]
LGTM, thanks for resolving the conflicts 💪

**Comment by PriyaKR** [comment]
Looks good to me.Did a general regression across articles,MAPs,PGL and frontpages.

**Pull request merged**